sequence_processing_pipeline/NuQCJob.py

```diff
@@ -401,6 +405,14 @@ def _generate_mmi_filter_cmds(self, working_dir):
 
         cores_to_allocate = int(self.cores_per_task / 2)
 
+        if len(self.additional_fastq_tags) > 0:
+            # add tags for known metadata types that fastq files may have
+            # been annotated with. Samtools will safely ignore tags that
+            # are not present.
+            tags = " -T %s" % ','.join(self.additional_fastq_tags)
```

Comment:
```suggestion
            tags = "-T %s" % ','.join(self.additional_fastq_tags)
```

Reply:
The odd spacing here is intentional. It prevents an extra space from being present when there are no tags. This makes the expected results for tests uniform.